src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/ParsersCommon.cs

```diff
@@ -435,11 +435,13 @@ void SkipDigits(bool signAllowed)
             }
             else
             {
-                string subString = _pathString.Substring(start, _curIndex - start);
-
                 try
                 {
-                    return System.Convert.ToDouble(subString, _formatProvider);
+#if PRESENTATION_CORE
```

Comment:
Could you use `NET` instead of PRESENTATION_CORE since it's .Net Core specific not PresentationCore project specific (In case in the future another .Net Core project references this file).
```suggestion
#if NET
```

Reply:
Thank you @ThomasGoulet73 

Could you review my code @stephentoub  ?

Reply:
> Could you review my code @stephentoub ?

Thanks, @lindexi.  As @ThomasGoulet73 noted, we don't need to duplicate all that code.  Really all that's needed is to swap the AsSpan for Substring.  I chose to duplicate the one line in the name of readability, but we shouldn't need to duplicate the whole block as your PR has.  I very much appreciate your efforts (and I hope others of your PRs can be merged), but I think we should close #5390 in favor of this one.

Reply:
> Could you use NET instead of PRESENTATION_CORE since it's .Net Core specific not PresentationCore project specific (In case in the future another .Net Core project references this file).

I agree with the reasoning why, but I chose PRESENTATION_CORE because it's already used with ifdefs multiple times in this file.  As such, I'll stick with the current ifdef.  If this file were to be included in other projects, I expect other ifdefs would also want to be revisited (even though this one is for a slightly different purpose).

Reply:
That's true but this means that PresentationBuildTasks, which multi-targets .Net Framework and .Net Core, will use Substring in both target frameworks when it could could use the span version for .Net Core.

Reply:
@stephentoub I agree with @ThomasGoulet73's argument, that we should use NET here instead of PRESENTATION_CORE flag as this code is not specific to PresentationCore.dll.